sky/templates/gcp-ray.yml.j2

```diff
@@ -143,8 +143,7 @@ setup_commands:
     pip3 install --upgrade google-api-python-client;
   {%- endif %}
     (pip3 list | grep ray | grep {{ray_version}} 2>&1 > /dev/null || pip3 install -U ray[default]=={{ray_version}}) && mkdir -p ~/sky_workdir && mkdir -p ~/.sky/sky_app;
-    pip3 uninstall skypilot -y &> /dev/null;
-    pip3 install "$(echo {{sky_remote_path}}/skypilot-{{sky_version}}*.whl)[gcp]" || exit 1;
+    [ "$(cat ~/.sky/current_sky_wheel_hash)" == "{{sky_wheel_hash}}" ] || (pip3 uninstall skypilot -y; pip3 install "$(echo {{sky_remote_path}}/skypilot-{{sky_version}}*.whl)[aws]" || exit 1; echo "{{sky_wheel_hash}}" > ~/.sky/current_sky_wheel_hash);
```

Comment:
1. `[aws]` -> `[gcp]` (our smoke tests should've caught this?)
2. consider moving the hash file to `~/.sky/sky_wheels/`? the root .sky dir is getting populous

Reply:
Great catch! Weirdly, our smoke tests does not catch the error. Probably because our launched cluster did not call anything in those dependencies.